src/templates/all/.make/markdown.mk

```diff
@@ -3,13 +3,18 @@
 # File origin: https://github.com/Arrow-air/tf-github/tree/main/src/templates/all/.make/markdown.mk
 
 MARKDOWN_FILES ?= $(shell find . -type f -iname '*md' ! -iwholename "*./node_modules/*" ! -path "./build" ! -iwholename "*.terraform*" ! -iwholename "*.cargo/*")
+LINK_CHECKER_JSON ?= .link-checker.config.json
 
 .help-markdown:
 	@echo ""
 	@echo "$(SMUL)$(BOLD)$(GREEN)Markdown$(SGR0)"
 	@echo "  $(BOLD)md-test-links$(SGR0)   -- Run markdown-link-check on all markdown files to catch dead links."
 
 md-test-links:
+ifeq ("$(wildcard $(LINK_CHECKER_JSON))","")
+	@echo "$(YELLOW)No $(LINK_CHECKER_JSON) found, creating...$(SGR0)"
+	echo -e "{\n}\n" > $(LINK_CHECKER_JSON)
+endif
```

Comment:
I think you want to move this block down actually, so we first check if we need to run and then check if the config file exists.
This way, there won't be a link checker file created if no markdown files are present in the repository.

Reply:
I fix: https://github.com/Arrow-air/tf-github/pull/42/commits/df4c72f5a7fc32f58cf29b21bbf8a90e43db689d